atlasdb-workload-server-distribution/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.palantir.sls-java-service-distribution'
 apply plugin: 'com.palantir.external-publish-dist'
 apply from: "../gradle/shared.gradle"
+apply from: "../gradle/tests.gradle"
```

Comment:
out of scope for this PR, though it does seem very weird that tests isn't a part of the shared.gradle